facia/public/humans.txt

```diff
@@ -127,7 +127,6 @@ Twitter: @mchv
 
 Developer: Nicolas Long
 Twitter: @room271
-Website: http://room271.net
```

Comment:
Oh @nicl 😢

Reply:
I confirmed it with him. The URL contains some malware and he doesn't own it anymore :)